lib/rubocop/cop/layout/multiline_block_layout.rb

```diff
@@ -94,13 +94,24 @@ def args_on_beginning_line?(node)
             node.loc.begin.line == node.arguments.loc.last_line
         end
 
+        # rubocop:disable Metrics/AbcSize
```

Comment:
Did you consider making `needed_length` a method to get the `AbcSize` down? E.g.,

```ruby
needed_length_for_args(node) > max_line_length
```